sdk/cosmos/azure-cosmos/azure/cosmos/documents.py

```diff
@@ -442,3 +443,11 @@ class _DistinctType(object):
     NoneType = "None"
     Ordered = "Ordered"
     Unordered = "Unordered"
+
+class PatchOperationType(object):
```

Comment:
I don't think I'd expose this - these are not particularly complex strings, so it would be longer for me to import and type our the object reference than to simply type the string. If we want the intellisense experience, we might get more value out of providing some kind of TypedDict type hint here?
e.g.
```python
class PatchOperation(TypedDict):
    op: Literal["add", "remove", "replace", "set", "incr", "move"]
    path: str
    from: NotRequired[str]
    value: NotRequired[Any]  # Maybe this could be a union of primary types

patch_operations: List[PatchOperation]
```

Reply:
yeah I agree, I added the class with the options based on a user request in one of the issues but your use of TypedDict definitely seems to make more sense in that regard - I'm all for either option, like you said it's not like it's complex strings and the samples show what is required for the body of each operation as well